werkzeug/test.py

```diff
@@ -329,6 +329,8 @@ def __init__(self, path='/', base_url=None, query_string=None,
         if data:
             if input_stream is not None:
                 raise TypeError('can\'t provide input stream and data')
+            if hasattr(data, 'seek'):
```

Comment:
![doc](https://cloud.githubusercontent.com/assets/49379/19001165/07bb3246-8714-11e6-8d9d-a8b9501d1e2b.png)
The three bullets use 4 space indentation and the rendering looks good to me.